subsys/bluetooth/crypto/bt_crypto.c

```diff
@@ -7,12 +7,12 @@
 
 #include <zephyr/sys/byteorder.h>
 
-#include <bt_crypto.h>
 #include <tinycrypt/cmac_mode.h>
 #include <tinycrypt/constants.h>
 
 #define BT_DBG_ENABLED	IS_ENABLED(CONFIG_BT_DEBUG_CRYPTO)
-#define LOG_MODULE_NAME bt_crypto_toolbox
+#define LOG_MODULE_NAME bt_crypto
+#include "crypto/bt_crypto.h"
```

Comment:
should be just: `bt_crypto.h`, as this header is in same folder as the source.
```suggestion
#include "bt_crypto.h"
```

Reply:
Sure, will fix.